src/Microsoft.PowerShell.SDK/Microsoft.PowerShell.SDK.csproj

```diff
@@ -16,25 +16,21 @@
 
   <ItemGroup>
     <!-- This section is to force the version of non-direct dependencies -->
-    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="7.0.0" />
-    <PackageReference Include="Microsoft.Extensions.ObjectPool" Version="7.0.3" />
-    <PackageReference Include="Microsoft.Win32.Registry" Version="5.0.0" />
-    <PackageReference Include="System.Web.Services.Description" Version="4.10.0" />
```

Comment:
Why are they removed?

Reply:
These are transitive dependencies, we do not need to add those anymore. They ship with the SDK and the latest version is auto-included.

Reply:
That's good to know, thanks!